packages/pubsub/src/index.js

```diff
@@ -717,17 +714,8 @@ PubSub.prototype.snapshot = function(name) {
  * @param {string=} name - The name of the subscription. If a name is not
  *     provided, a random subscription name will be generated.
```

Comment:
I would ask you disregard any documentation that looks off, it's on my TODO list to update/write the docs once we've reached an agreement. This particular piece of documentation is old and there is no code in the redesign that would generate a subscription name for the user.